spec/policies/user_policy_spec.rb

```diff
@@ -7,27 +7,30 @@
 
   context 'student' do
     let(:user) { FactoryGirl.build_stubbed(:user, role: 'student') }
-    permissions :show?, :update? do
+    permissions :show?, :update?, :edit?, :edit_intent?, :update_intent? do
```

Comment:
nope, we also forgot it for suites and possibly others. I'll open up a quick issue for it